packages/mgt-components/src/components/mgt-agenda/mgt-agenda.ts

```diff
@@ -384,9 +397,38 @@ export class MgtAgenda extends MgtTemplatedComponent {
    * @memberof MgtAgenda
    */
   protected renderTitle(event: MicrosoftGraph.Event): TemplateResult {
-    return html`
-      <div aria-label=${event.subject} class="event-subject">${event.subject}</div>
+    let eventDescription = event?.bodyPreview ? event.bodyPreview.slice(0, 100) : '';
+    const hasDescription = eventDescription !== '';
+
+    const eventSubjectClasses = {
+      'event-subject': true,
+      narrow: this._isNarrow
+    };
+
+    eventDescription = eventDescription.split(' ').slice(0, -1).join(' ') + '...';
+
+    const hasDescriptionDiv = html`
+      <div
+        aria-describedby="tooltip-${event.id}"
+        class="${classMap(eventSubjectClasses)}"
+        id=${event.id}>
+          ${event.subject}
+      </div>
+      <fluent-tooltip
+        id="tooltip-${event.id}"
+        position="right"
```

Comment:
There's something still off with the positioning of this tooltip as it's falling outside of the page on the left and looks wrong

Reply:
Check it at the moment if there's a change from what you saw earlier?

Reply:
Hmm.. shows like this locally
![image](https://user-images.githubusercontent.com/8081536/212688022-1a9c8a54-e037-4356-a2dd-168502939f4f.png)

But on the PR playground its off. Investigating.